pkg/xray/reports.go

```diff
@@ -525,15 +525,15 @@ func unpackViolationsFilters(filter *schema.Set) *Filters {
 		filters.PolicyNames = util.CastToStringArr(m["policy_names"].(*schema.Set).List())
 		filters.Severities = util.CastToStringArr(m["severities"].(*schema.Set).List())
 
-		if m["updated"] != nil {
+		if m["updated"].(*schema.Set).Len() > 0 {
```

Comment:
Do we still need to check for nil first? e.g. `if m["updated"] != nil && m["updated"].(*schema.Set).Len() > 0`

Reply:
It's never actually a `nil`, I checked. That was a reason it crashed and user created a ticket.